inc/zoo/swar/math.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include "SWAR.h"
+#include <cstddef>
+#include <cstdint>
+
+namespace zoo::math {
+
+template <typename IntegerType = size_t>
+constexpr static
+std::enable_if_t<std::is_integral_v<IntegerType>, bool>
+is_power_of_two(IntegerType x) noexcept {
```

Comment:
Why `snake_case` instead of the normal `camelCase` and `DromedaryCamelCase` customary in the rest of `zoo` for these things?

Reply:
habit, let's move to camelCase to be consistent. will start email thread about case to ask if you have strong preference to avoid polluting PR with unrelated discussion.